examples/2-with-_app.js/pages/_app.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { withUrqlClient } from 'next-urql';
+
+interface IApp {
+  Component: React.FC;
+  pageProps: Record<string, any>;
+}
+
+const App: React.FC<IApp> = ({ Component, pageProps }) => (
```

Comment:
If I'm not mistaken, I believe `next` has a type specifically for props handed to `_app.js` components: https://github.com/zeit/next.js/blob/58b2d9e208c56cee0dab2eff287d2a19cb0e6de1/packages/next/next-server/lib/utils.ts#L129-L135. It's not super precise, but let's use there type signature here:

```suggestion
const App: React.FC<AppPropsType> = ({ Component, pageProps }) => (
```

Reply:
Oh cool, I didn't see that! Updating now